### DISCUSSION TITLE
Siembol Demo Lab at DEF CON 29 - August 2021.

### DISCUSSION THREAD

**Discussion by c-rindi** [G-Research/siembol#120] [Events]
Siembol will be presenting at DEF CON 29!

https://forum.defcon.org/node/237250

Come hear Marian Novotny talk about "Anti-Malware for the Cloud".

Stay tuned for the exact date/time of the Demo Lab, or come join us in Las Vegas.
Conference dates are Aug 5-8 at Bally's/Paris hotels

We will also be presenting at Black Hat a few days prior.

Any conference video will be posted in this channel afterwards.

**Comment by c-rindi** [comment]
Session will be on Friday at 12pm noon PST, and will be a virtual presentation.

More details [here](https://forum.defcon.org/node/237250) and [here](https://forum.defcon.org/node/238866).

Join the conversation on Discord Channel: #dl-siembol

**Reply by mariannovotny** [reply]
DEF CON Discord server is [here](https://discord.com/invite/defcon)
